tmt/steps/execute/internal.py

```diff
@@ -318,11 +334,11 @@ def _run_tests(self, guest, extra_environment=None):
         self.debug("Pull the plan data directory.", level=2)
         guest.pull(source=self.step.plan.data_directory)
 
-    def results(self):
+    def results(self) -> List[Result]:
         """ Return test results """
         return self._results
 
-    def requires(self):
+    def requires(self) -> List[str]:
         """ Return list of required packages """
         # FIXME Remove when we drop support for the old execution methods
         return ['beakerlib'] if self.step._framework == 'beakerlib' else []
```

Comment:
`tmt/steps/execute/internal.py:344: error: "Step" has no attribute "_framework"`
I checked the class and it's parent classes and it really doesn't have `_framework`. The class `Execute` has it though

Reply:
@happz might be this relates to some of the issues you filled?

Reply:
Yeah, it's related to #1373 & co., `self.step` here is just `Step` with no extra detail, while to humans, it's obvious it can ever be `Execute.

Please, add a TODO item here, refering to the issue, and in the meantime `cast(tmt.steps.execute.Execute, self.step)._framework` should help.

Reply:
Done. Thanks.